.github/workflows/image-push.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     name: Push image to quay.io
-    if: ${{ contains(github.event.head_commit.message, 'narayana-bom') || contains(github.event.head_commit.message, 'quarkus.platform.version') || github.event_name == 'workflow_dispatch' }}
+    if: ${{ contains(github.event.head_commit.message, 'narayana-bom') || contains(github.event.head_commit.message, 'Dockerfile') || contains(github.event.head_commit.message, 'quarkus.platform.version') || github.event_name == 'workflow_dispatch' }}
```

Comment:
Updated the image-push action to be triggered when the dockerfile is updated

Reply:
No, remove this. We tag the image as {narayana-version}-{quarkus-version}, so if you try to deploy image only on docker change that will try to push the same tag again and it will always fail. Don't worry about this, there is Quarkus or Narayana release often enough.